data/publiccloud/cloud-init.yaml

```diff
@@ -0,0 +1,40 @@
+#cloud-config
```

Comment:
Other tests, like `data/jeos/g_cloudinit.cfg`, also play with `bootcmd:`

Reply:
Added. Feel free to check.